pkg/controller/rpaasinstance/rpaasinstance_controller.go

```diff
@@ -41,9 +42,41 @@ import (
 )
 
 const (
-	defaultConfigHistoryLimit = 10
-
+	defaultConfigHistoryLimit     = 10
+	defaultCacheHeaterCronImage   = "bitnami/kubectl:latest"
+	defaultCacheHeaterSchedule    = "* * * * *"
 	defaultPortAllocationResource = "default"
+	volumeTeamLabel               = "tsuru.io/volume-name"
+
+	cacheHeaterCronJobSuffix = "-heater-cron-job"
+	cacheHeaterVolumeSuffix  = "-heater-volume"
+
+	rsyncCommandPodToPVC = "rsync -avz --recursive --delete --temp-dir=/var/cache/cache-heater/temp /var/cache/nginx/rpaas/nginx /var/cache/cache-heater"
+	rsyncCommandPVCToPod = "rsync -avz --recursive --delete --temp-dir=/var/cache/nginx/rpaas/nginx_tmp /var/cache/cache-heater/nginx /var/cache/nginx/rpaas"
+)
+
+var (
+	defaultCacheHeaterCmdPodToPVC = []string{
+		"/bin/bash",
+		"-c",
+		`
+pods=($(kubectl -n ${SERVICE_NAME} get pod -l rpaas.extensions.tsuru.io/service-name=${SERVICE_NAME} -l rpaas.extensions.tsuru.io/instance-name=${INSTANCE_NAME} --field-selector status.phase=Running -o=jsonpath='{.items[*].metadata.name}'));
+for pod in ${pods[@]}; do
+	kubectl -n ${SERVICE_NAME} exec ${pod} -- ${POD_CMD};
+	if [[ $? == 0 ]]; then
+		exit 0;
+	fi
+done
+`}
+
+	defaultCacheHeaterCmdPVCToPod = []string{
+		"/bin/bash",
+		"-c",
+		`
+mkdir -p /var/cache/cache-heater/temp;
+mkdir -p /var/cache/nginx/rpaas/nginx_tmp;
+bash -c "${POD_CMD}"
```

Comment:
I think there's no need for another shell here. This could simply be:
```suggestion
${POD_CMD}
```